README.md

```diff
@@ -2,14 +2,15 @@
 [![verify](https://github.com/checkpoint-restore/go-criu/workflows/verify/badge.svg?branch=master)](https://github.com/checkpoint-restore/go-criu/actions?query=workflow%3Averify)
 [![Go Reference](https://pkg.go.dev/badge/github.com/checkpoint-restore/go-criu.svg)](https://pkg.go.dev/github.com/checkpoint-restore/go-criu)
 
-## go-criu -- Go bindings for CRIU
+# go-criu -- Go bindings for CRIU
```

Comment:
It would be good if we can use the same format as the README in the [criu repository](https://raw.githubusercontent.com/checkpoint-restore/criu/criu-dev/README.md) (`##` instead of `#`)